pandas-stubs/_version.pyi

```diff
@@ -4,4 +4,4 @@ version_json: str = ...
 
 def get_versions(): ...
 
-_stub_version = Literal["1.5.0.220926"]
+_stub_version: Literal["1.5.0.220926"]
```

Comment:
Not sure why this is needed.

If we assign Literal as a value, we need a TypeAlias, but the type alias is unused  (private type aliases need to be used) which triggers another check from flake8-pyi.

Reply:
Microsoft is copying the stubs over to the VS Code pylance distribution, so by having this, we can tell what version they copied over.